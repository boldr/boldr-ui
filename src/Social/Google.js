/* @flow */
import React from 'react';
import Icon from '../Icons';

type Props = {
  href: ?string,
  size: string,
};

const Google = (props: Props) => {
  if (props.href) {
    return (
      <a href={props.href} target="_blank">
        <Icon kind="google" color="#1c3050" size={props.size} />
      </a>
    );
  } else {
    return <Icon kind="google" color="#1c3050" size={props.size} />;
  }
};

const defaultProps = {
  size: '24px',
};

Google.defaultProps = defaultProps;

export default Google;
