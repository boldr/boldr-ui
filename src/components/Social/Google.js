/* @flow */
import React from 'react';
import Icon from '../Icons';

type Props = {
  href: ?String,
  size: Number,
};

const Google = (props: Props) => {
  if (props.href) {
    return (
      <a href={props.href} target="_blank">
        <Icon kind="google" color="#1c3050" size={props.size} />
      </a>
    );
  } else {
    return <Icon kind="google" color="#1c3050" size={props.size} />;
  }
};

Google.defaultProps = {
  size: 24,
};
export default Google;
