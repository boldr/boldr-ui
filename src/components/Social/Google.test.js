import React from 'react';
import { shallow, mount } from 'enzyme';
import Google from './Google';

test('<Google />', () => {
  const wrapper = shallow(<Google href="a" />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
