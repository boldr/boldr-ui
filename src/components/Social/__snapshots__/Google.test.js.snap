// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Google /> 1`] = `
<a
  href="a"
  target="_blank"
>
  <Icon
    color="#1c3050"
    kind="google"
    size={24}
  />
</a>
`;
